pages/loginPage.tsx

```diff
@@ -0,0 +1,134 @@
+import Image from "next/image";
+import React, { ChangeEvent, useState } from "react";
+import Link from "next/link";
+import ErrorModalHandler from "../components/errorModalHandler";
+import { useRouter } from "next/router";
+import { loginPage } from "../public/locales/fa.json";
+//import { en } from "../public/locales/en";
+
+type PropsList = {
+  handleListCompany: () => void;
+  handleLoginPage: () => void;
+};
+
+const LoginPage: React.FC<PropsList> = (props) => {
+  const [testName, setTestName] = useState<string | number>();
```

Comment:
- Why names should have ```number``` type?
- Why ```testName``` and ```setTestName```? use a good variable name
```suggestion
  const [name, setName] = useState<string>();
```